pkgs/development/libraries/openimagedenoise/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, cmake, tbb, git, python }:
+
+stdenv.mkDerivation rec {
+  name = "openimagedenoise";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "OpenImageDenoise";
+    repo = "oidn";
+    rev = "v${version}";
+    sha256 = "032s7vablqnmrcc4xf2c94kwj0kbcd64bram10g0yc42fg0a3r9m";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ cmake tbb git python ];
```

Comment:
In this case, `tbb` is used as a library and should go to `buildInputs`. Also, you don't need `git` here.